### DISCUSSION TITLE
Add metrics from 'states' API  endpoint

### DISCUSSION THREAD

**Discussion by pbrinette** [acesyde/hassio_mylight_integration#43] [Ideas]
Thanks @acesyde for your wonderful works.

I've noticed that the API have a 'states' endpoint (`https://myhome.mylight-systems.com/api/states`) with some interesting metrics.

It would be great to report informations about the virtual battery  :

The battery capacity is described in the `devices` endpoint : 

```
    },
    {
      "id": "YPbhyA2XXXXX2J4z_msb",
      "name": "Battery 1",
      "type": "bat",
      "batteryCapacity": 600,
      "batteryNumberOfBoxes": 1,
...
```

And the actual capacity informations are defined in the `states` endpoint.

```
...
       {
          "sensorId": "YPbhyA2XXXXX2J4z_msb-soc",
          "measure": {
            "value": 523140069.59999996,
            "type": "soc",
            "unit": "Ws",
            "date": "2023-05-05 16:45:00"
          }
        }
```

What do you think about that ?

**Comment by acesyde** [comment]
I have my battery activated since 3 day and I have found exactly the same endpoint yesterday to retrieve the battery current state, I will implement it in a couple of days or if you want you can made a PR 👍

**Comment by acesyde** [comment]
@pbrinette you can try with `0.0.5-beta.1` a new sensor is available ;)

**Comment by pbrinette** [comment]
Thanks @acesyde . I've asked some questions on the HACS French forum page related to your integration.

**Reply by pbrinette** [reply]
I've tested the last release. The battery information are correctly handled.
